code/__HELPERS/game.dm

```diff
@@ -324,6 +324,20 @@
 		else if(isliving(M.current))
 			return M.current.stat != DEAD
 	return FALSE
+	
+/**
+  * Exiled check
+  * 
+  * Checks if the current body of the mind has an exile implant and is currently in 
+  * an away mission. Returns FALSE if any of those conditions aren't met.
+  */ 
+/proc/considered_exiled(datum/mind/M)
+	if(!ishuman(M?.current))
+		return FALSE
+	else
+		for(var/obj/item/implant/I in M.current.implants)
+			if(I == /obj/item/implant/exile && M.current.onAwayMission())

```

Comment:
```suggestion
			if(istype(I, /obj/item/implant/exile) && M.current.onAwayMission())
```

Reply:
This is assuming it's a list of instances which I'm prety sure it is. Otherwise you'd do
```suggestion
			if(ispath(I, /obj/item/implant/exile) && M.current.onAwayMission())
```